tests/backup/backup_upgrade_test.go

```diff
@@ -202,3 +203,509 @@ var _ = Describe("{StorkUpgradeWithBackup}", func() {
+			namespaceMapping := make(map[string]string, 0)
+			err = CreateRestore(restoreName, firstAllNSScheduleBackupName, namespaceMapping, SourceClusterName, orgID, ctx, make(map[string]string, 0))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restoration [%s] of first all namespaces schedule backup [%s] in cluster [%s]", restoreName, firstAllNSScheduleBackupName, restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		// By the time the next allNS schedule backups are taken, the px-backup upgrade would have been completed
+		Step("Restore next all namespaces schedule backup", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+			nextScheduleBackupName, err := GetNextScheduleBackupName(allNSScheduleName, time.Duration(intervalInMins), ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching next schedule backup name of schedule named [%s]", allNSScheduleName))
+			log.InfoD("Next schedule backup name [%s]", nextScheduleBackupName)
+			restoreName := fmt.Sprintf("%s-%s", "test-restore-all-ns", RandomString(4))
+			log.InfoD("Restoring next all namespaces schedule backup [%s] in cluster [%s] with restore [%s]", nextScheduleBackupName, SourceClusterName, restoreName)
+			namespaceMapping := make(map[string]string, 0)
+			err = CreateRestore(restoreName, nextScheduleBackupName, namespaceMapping, SourceClusterName, orgID, ctx, make(map[string]string, 0))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restoration [%s] of next all namespaces schedule backup [%s] in cluster [%s]", restoreName, nextScheduleBackupName, restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+	})
+	JustAfterEach(func() {
+		allContexts := append(srcClusterContexts, destClusterContexts...)
+		defer EndPxBackupTorpedoTest(allContexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		log.InfoD("Deleting single namespace Backup schedules %s", singleNSScheduleNames)
+		deleteSingleNSScheduleTask := func(scheduleName string) {
+			scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+			log.FailOnError(err, "Fetching uid of single namespace backup schedule [%s]", scheduleName)
+			log.Infof("Single namespace backup schedule [%s] uid: [%s]", scheduleName, scheduleUid)
+			log.InfoD("Deleting single namespace backup schedule [%s]", scheduleName)
+			err = DeleteSchedule(scheduleName, SourceClusterName, orgID, ctx)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of backup schedule [%s]", scheduleName))
+		}
+		_ = TaskHandler(singleNSScheduleNames, deleteSingleNSScheduleTask, Parallel)
+		log.InfoD("Deleting all namespaces Backup schedule %s", allNSScheduleName)
+		scheduleUid, err := GetScheduleUID(allNSScheduleName, orgID, ctx)
```

Comment:
We no more need to fetch schedule id, This has been handled inside DeleteSchedule now.